Python/Product/PythonTools/PythonTools/WebProject/PythonWebLauncher.cs

```diff
@@ -108,16 +93,10 @@ LaunchConfiguration defaultConfig
             if (debug) {
                 _pyService.Logger.LogEvent(Logging.PythonLogEvent.Launch, 1);
 
+                config.LaunchOptions[PythonConstants.WebBrowserUrlSetting] = url.AbsoluteUri;
                 using (var dsi = DebugLaunchHelper.CreateDebugTargetInfo(_serviceProvider, config)) {
                     dsi.Launch();
                 }
-
-                var debugger = (IVsDebugger)_serviceProvider.GetService(typeof(SVsShellDebugger));
```

Comment:
It's handled by the debugger when the web browser property is passed through, which it is in this case.